utils/src/main/java/com/cloud/utils/fsm/StateMachine2.java

```diff
@@ -171,15 +179,22 @@ public String toString() {
 
       private List<Impact> impacts;
 
+      private boolean alertFlag;
+
       public static enum Impact {
         USAGE
       }
 
       public Transition(S currentState, E event, S toState, List<Impact> impacts) {
+        this(currentState, event, toState, impacts, false);
+      }
+
+      public Transition(S currentState, E event, S toState, List<Impact> impacts, boolean alertFlag) {
         this.currentState = currentState;
```

Comment:
Old constructor just calls new constructor with False for the alertFlag. So we don't break any functionality using the previous constructor